src/phases/title-phase.ts

```diff
@@ -100,7 +100,7 @@ export class TitlePhase extends Phase {
               }
             }
           ];
-          if (this.scene.gameData.unlocks[Unlockables.SPLICED_ENDLESS_MODE]) {
+          if (this.scene.gameData.isUnlocked(Unlockables.SPLICED_ENDLESS_MODE)) {
```

Comment:
as you buffered `gameData` previously you can now do:

```suggestion
          if (gameData.isUnlocked(Unlockables.SPLICED_ENDLESS_MODE)) {
```